src/scripts/sssom_to_robot_template.py

```diff
@@ -0,0 +1,57 @@
+"""Convert SSSOM to ROBOT template."""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, Union
+
+import pandas as pd
+from sssom import MappingSetDataFrame
+from sssom.parsers import parse_sssom_table
+
+ROBOT_ROW = {
+    'subject_id': 'ID',
+    'subject_label': '',
+    'object_id': 'A oboInOwl:hasDbXref',
+    'equivalence': '>A oboInOwl:source',
+    'author_id': '>A oboInOwl:source SPLIT=|',
```

Comment:
```suggestion
    'author_id': '>AI oboInOwl:source SPLIT=|',
```

Orcids are IRIs:

xref: ICD10CM:N00-N99 {source="MONDO:obsoleteEquivalent", source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"}

Reply:
Understood! Thanks. Committing the suggestion